mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt

```diff
@@ -195,7 +196,9 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
 
         if (cached != null) {
             applyLoadedBitmap(cached)
-        } else {
+        } else if (url.newBuilder().removeAllQueryParameters("state").build() !=
+            lastRequest?.url?.newBuilder()?.removeAllQueryParameters("state")?.build()
```

Comment:
Maybe add a method `fun statelessUrlEquals(url: HttpUrl): Boolean` to `HttpRequest` and call it like `} else if (lastRequest?.statelessUrlEquals(url) != true) {`
?